### PR TITLE
Added capability to convert animation sequence to json string

### PR DESCRIPTION
**Pull request by anand-sundaram**
(Also abstracted the `UIColor`s  currently used into an enum called Colour)

`AnimationSequence` can be serialized into a JSON string so @advaypal can store it easily using realm.

- `getJSON() -> String`
- `AnimationSequence.getAnimationSequenceFromJSON(fromJSON: String) -> AnimationSequence`

### CODE REVIEW

File: MIDIchlorians/MIDIchlorians/AnimationBit.swift [hunk 1]
```diff
@@ -9,13 +9,51 @@
 import UIKit
 
 struct AnimationBit {
-    var colour: UIColor
+    var colour: Colour
     var row: Int
     var column: Int
 
-    init(colour: UIColor, row: Int, column: Int) {
+    static private let colourKey = "colour"
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians/AnimationBit.swift]: extract to config file?

Comment (ngzhian) [reply MIDIchlorians/MIDIchlorians/AnimationBit.swift]: same for `rowKey` and `columnKey`

File: MIDIchlorians/MIDIchlorians/AnimationSequence.swift [hunk 2]
```diff
@@ -45,4 +45,65 @@ class AnimationSequence {
         tickCounter += 1
         return array
     }
+
+    func getJSON() -> String? {
+        var arrayOfStrings = [[String]?]()
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians/AnimationSequence.swift]: maybe call this something like, animationSequence? Cos it's an animation sequence JSON in string form right?

**Review by ngzhian — CHANGES_REQUESTED** [review]

**Comment by anand-sundaram** [comment]
@ngzhian made some changes based on your review.

**Review by ngzhian — APPROVED** [review]

**Comment by ngzhian** [comment]
LGTM, so merged this in. @advaypal please merge this into your local branch and make changes based on this

**Pull request merged**